scripts/ec2/ls.coffee

```diff
@@ -118,38 +118,23 @@ handle_instances = (robot) ->
     instances_that_expired = instances.filter instance_has_expired
     instances_that_will_expire = instances.filter instance_will_expire_soon
 
-    msg_text_expired = extract_message(instances_that_expired, EXPIRED_MESSAGE)
     msg_text_expire_soon = extract_message(instances_that_will_expire, EXPIRES_SOON_MESSAGE)
-
-    msg_room(msg_text_expired)
-    msg_room(msg_text_expire_soon)
-
-    for user in _.values(robot.brain.data.users)
-      creator_email = user.email_address || "_DL_CFPB_Software_Delivery_Team@cfpb.gov"
-      user_id = user.id || 1
-
-      user_instances = _.filter(instances_that_will_expire, (this_instance)-> return get_instance_tag(this_instance, 'Creator') == creator_email)
-
-      if user_instances
-        msg_text_expire_soon = extract_message(user_instances, USER_EXPIRES_SOON_MESSAGE) + EXTEND_COMMAND
-
-    msg_room(msg_text_expired)
     msg_room(msg_text_expire_soon)
 
     for user in _.values(robot.brain.data.users)
-      creator_email = user.email_address || "_DL_CFPB_Software_Delivery_Team@cfpb.gov"
-      user_id = user.id || 1
+      creator_email = user.email_address
+      user_id = user.id
 
-      user_instances = _.filter(instances_that_will_expire, (this_instance)-> return get_instance_tag(this_instance, 'Creator') == creator_email)
+      user_instances = _.filter(instances_that_will_expire, (this_instance)-> return get_instance_tag(this_instance, 'Creator').toLowereCase() == creator_email.toLowerCase())
 
-      if user_instances
+      if user_instances.length
         msg_text_expire_soon = extract_message(user_instances, USER_EXPIRES_SOON_MESSAGE) + EXTEND_COMMAND
-        robot.send({user: user_id}, msg_text_expire_soon)
+        msg_room("@#{user.name}: #{msg_text_expire_soon}")
 
-    ec2.stopInstances {InstanceIds: _.pluck(instances_that_expired, 'InstanceId')}, (err, res) ->
+    instanceIdsToStop = _.pluck(instances_that_expired, 'InstanceId')
```

Comment:
agreed, by filtering them out of instances_that_expired either as part of the query to get expired instances, or immediately after fetching